tests/sentry/utils/performance_issues/test_m_n_plus_one_db_detector.py

```diff
@@ -37,7 +37,7 @@ def test_detects_parallel_m_n_plus_one(self):
             PerformanceProblem(
                 fingerprint="1-1011-6807a9d5bedb6fdb175b006448cddf8cdf18fbd8",
                 op="db",
-                type=PerformanceMNPlusOneDBQueriesGroupType,
+                type=PerformanceNPlusOneGroupType,
```

Comment:
shouldn't this be MNPlusOne in the mn detector test?

Reply:
Based on how I understood the PR description, this PR changes MN+1 to be a case of N+1, rather then its own thing.

But its still a good thing to mention, because this type is used to control the UI span evidence section, which is relevant if we want the UI to differ in anyway based on the type alone. (if you can change the UI based on the offending spans then it should be fine)

Reply:
No - we decided to release these as N+1 issues rather than giving them their own type. In other words, this new detector is a way to detect more problems for the existing issue.